app/static/app/js/components/ExportAssetPanel.jsx

```diff
@@ -9,7 +9,7 @@ import Workers from '../classes/Workers';
 
 export default class ExportAssetPanel extends React.Component {
   static defaultProps = {
-      exportFormats: ["gtiff-rgb", "gtiff", "jpg", "png", "kmz"],
+      exportFormats: ["gtiff-rgb", "gtiff", "jpg", "png", "kmz","gpkg"],
```

Comment:
Mm, I think the intent here was to allow the download of the cutline, but this isn't the place where to add it.